Do not try to get version information from git. It will not work because we are
using a tarball not a git clone.
https://savannah.nongnu.org/bugs/index.php?65943
--- cmake/get_git_version.sh.orig	2019-12-20 07:13:39.000000000 -0600
+++ cmake/get_git_version.sh	2024-07-03 12:28:33.000000000 -0500
@@ -1,5 +1,5 @@
 #!/bin/sh
-if which git > /dev/null 2> /dev/null; then
+if false; then
   H=$(git log --pretty=format:'%h' -n 1)
   D=$(git diff --quiet --exit-code || echo "+")
   T=$(git describe --abbrev=0)
